INSTALL.adoc

```diff
@@ -198,3 +198,71 @@ and sanity checks that could help you pinpoint the problem.
   requirements for mathematical accuracy, numerical precision, and proper
   handling of mathematical corner cases and error conditions may need to
   consider running their code on a platform with better C99 support.
+
+== (experimental) Building a cross compiler
+
+A cross compiler is a compiler that runs on some machine, named the
+_host_, but generate code for different machine, named the _target_. To
```

Comment:
```suggestion
_host_, but generates code for a different machine, named the _target_. To
```
(you might want to replace `machine' with `architecture' in this sentence as well)

Reply:
I used the term “machine” as in `gcc -dumpmachine`, thinking “architecture” is often interpreted as only the `x86_64` in `x86_64-linux-gnu`.